packages/driver/src/cy/big_value.ts

```diff
@@ -0,0 +1,149 @@
+import _ from 'lodash'
```

Comment:
NOTE: we might want just override chai.util.objDisplay. We do that in the firefox PR to avoid an error when the object contains a cross-origin object. This will need some refactoring after the firefox PR